inspirehep/accounts/api.py

```diff
@@ -19,3 +19,10 @@ def check_if_is_logged_in(*args, **kwargs):
         return func(*args, **kwargs)
 
     return check_if_is_logged_in
+
+
+def is_superuser_or_cataloger_logged_in():
+    if current_user.is_authenticated:
+        user_roles = [role.name for role in current_user.roles]
+        return any(role in user_roles for role in ("superuser", "cataloger"))
```

Comment:
this is a set intersection:
```python
user_roles = {role.name for role in current_user.roles}
return user_roles & {"superuser", "cataloger"}
```

Reply:
i did with `set` but looked more clear this way, since it's always 1 or 2 items doesn't matter performance wise.